components/admin_console/admin_sidebar/admin_sidebar.jsx

```diff
@@ -88,6 +89,26 @@ export default class AdminSidebar extends React.Component {
         }
 
         this.updateTitle();
+
+        if (!Utils.isMobile()) {
+            $('.admin-sidebar .nav-pills__container').perfectScrollbar({
+                suppressScrollX: true,
+            });
+        }
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.idx != null &&
+            (!isEqual(this.props.plugins, prevProps.plugins) ||
+                !isEqual(this.props.adminDefinition, prevProps.adminDefinition))) {
+            this.idx = generateIndex(this.props.adminDefinition, this.context.intl, this.props.plugins);
+        }
+
+        if (!Utils.isMobile()) {
+            $('.admin-sidebar .nav-pills__container').perfectScrollbar({
+                suppressScrollX: true,
```

Comment:
Same comment as above regarding this scrollbar change

Reply:
Not sure how these came in..Thanks for spotting. Removed 👍